app/views/tasks/_task_table.html.erb

```diff
@@ -15,7 +15,11 @@
 
   <tbody>
     <% tasks.each do |task| %>
+      <% if task.overdue? %>
+        <tr class="priority<%= task.priority %> overdue">
+      <% else %>
       <tr>
```

Comment:
This is the nitpickiest of nitpicks, but would you mind tabbing this `<tr>` in to match the other one?